plenoptic/synthesize/eigendistortion.py

```diff
@@ -351,11 +351,11 @@ def _synthesize_power(self,
             Fv = fisher_info_matrix_vector_product(y, x, v, _dummy_vec)
             Fv = Fv - shift * v  # minor component
 
-            v_new = torch.qr(Fv)[0] if k > 1 else Fv
+            v_new = torch.qr(Fv)[0]  # (ortho)normalize vector(s)
 
             lmbda_new = fisher_info_matrix_eigenvalue(y, x, v_new, _dummy_vec)
 
-            d_lambda = (lmbda.sum() - lmbda_new.sum()).norm()  # stability of eigenspace
+            d_lambda = (lmbda - lmbda_new).norm()  # stability of eigenspace
```

Comment:
Why make this change?

Reply:
I experimented with both and I think the new line is more precise. Each indiv eigenvec must converge to some value rather than total sum of eigenvalues converging.